test/foundry-tests/PoolManager.t.sol

```diff
@@ -227,6 +230,65 @@ contract PoolManagerTest is Test, Deployers, TokenFixture, GasSnapshot, IERC1155
         assertEq(slot0.sqrtPriceX96, sqrtPriceX96);
     }
 
+    function testPoolManagerInitializeSucceedsWithRevertingFeeController(uint160 sqrtPriceX96) public {
```

Comment:
not super sure we need this test but wanted to make sure that hook fees were not affected